docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: '3'
 services:
   postgres:
     image: 'postgres'
+    platform: linux/amd64
```

Comment:
@cesumilo could you explain how this related to arm architecture, and why it was added? I see amd64 here?

Reply:
@lehni As I'm developing on an Macbook Pro M2 (ARM architecture), I need to specify in the `docker-compose.yml` file [what platform I'm targeting when building the containers](https://docs.docker.com/compose/compose-file/05-services/#platform) so that docker knows how to run it later. This addition shouldn't affect other contributors but it'll make ARM-dev's life like me much easier 😄 .

Reply:
I see. I am curious, ` linux/arm64` doesn't work for you?

Reply:
I think this should be done separately thought, as it does not relate to the topic.

Reply:
Ok I'll create a separate PR for this and a dedicated issue 🆗 
When I use `linux/arm64` or no platform at all, here is the issue:
```bash
[+] Running 1/2
 ⠙ mysql Pulling                                                                                                                                                                                                                                                                                                                                                                                                                               1.1s 
 ✘ postgres Error                                                                                                                                                                                                                                                                                                                                                                                                                              1.1s 
no matching manifest for linux/arm64 in the manifest list entries
```

Reply:
@lehni Moved 🆗